include/deal.II/dofs/dof_accessor.templates.h

```diff
@@ -2900,6 +2920,28 @@ DoFCellAccessor<dimension_, space_dimension_, level_dof_access>::
 
 
 
+template <int dimension_, int space_dimension_, bool level_dof_access>
+inline unsigned int
+DoFCellAccessor<dimension_, space_dimension_, level_dof_access>::
+  dominated_future_fe_on_children() const
+{
+  std::set<unsigned int> future_fe_indices_children;
+  for (const auto &child : this->child_iterators())
+    future_fe_indices_children.insert(child->future_fe_index());
+  Assert(!future_fe_indices_children.empty(), ExcInternalError());
```

Comment:
> This function can only be called on direct parent cells, i.e., non-active cells whose children are all active.

Aren't here some asserts missing?

Reply:
These assertions will be made when calling `cell->future_fe_index()`.

I agree that we should make these assertions here as well. Will comply!

Note: This will be part of a later PR.